### PR TITLE
Merge master to release/3.0.1xx for preview6

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

**Comment by livarcocc** [comment]
Clean merge?

File: eng/Versions.props [hunk 1]
```diff
@@ -18,18 +18,17 @@
...
     <PlatformAbstractionsVersion>2.0.0</PlatformAbstractionsVersion>
     <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
     <SystemReflectionMetadataVersion>1.5.0</SystemReflectionMetadataVersion>
-    <MicrosoftDotNetSignToolVersion>1.0.0-beta.19218.7</MicrosoftDotNetSignToolVersion>
+    <MicrosoftDotNetSignToolVersion>1.0.0-beta.19279.5</MicrosoftDotNetSignToolVersion>
   </PropertyGroup>
   <!-- Get .NET Framework reference assemblies from NuGet packages -->
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
-    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-5</MicrosoftNetFrameworkReferenceAssembliesVersion>
```
Comment (livarcocc) [inline eng/Versions.props]: @dsplaisted how are we referencing this version now?

Comment (dsplaisted) [reply eng/Versions.props]: It's [defined by arcade](https://github.com/dotnet/arcade/blob/ed3510aaa6610514c484e23a4b7f17963bf16e0c/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L83).  I'm not sure why we had it pinned to an earlier version, maybe we were just using the same property name before arcade had this support built in.

Comment (livarcocc) [reply eng/Versions.props]: Cool.

**Pull request merged**